### DISCUSSION TITLE
Change Quiet Public description to clarify function and remove misleading algorithm reference

### DISCUSSION THREAD

**Discussion by FediVideos** [mastodon/mastodon#30425] [General]
### Steps to reproduce the problem

1. Log in through web interface on instance running latest pre-release
2. Click visiblity icon on post editing box
3. Observe "quiet public" is described as "fewer algorithmic fanfares"
...

### Expected behaviour

It should say "Hidden from search and trends"

### Actual behaviour

It says "Fewer algorithmic fanfares"

### Detailed description

Description on Quiet Public should be changed to "Hidden from search and trends". This would explain exactly what it does without introducing misleading wording. (Technically Quiet Public also hides posts from Federated and Local timelines, but this won't fit on the space available for the description and search/trends are more relevant to new users.)

Most non-technical people understand "algorithm" to mean Facebook-style stuff that hides what you are trying to see and pushes stuff at you that you don't want to see. Quiet Public/Unlisted visibility doesn't do any of this at all, Mastodon in general doesn't do any of this at all, so it is best to avoid the word.

### Mastodon instance

mastodon.social

### Mastodon version

v4.3.0-nightly.2024-04-30

### Browser name and version

Firefox

### Operating system

Ubuntu

**Comment by Gargron** [comment]
Trends are algorithms.

**Comment by FediVideos** [comment]
> Trends are algorithms.

Technically yes, but not really in common language, at least not in English.

If English-speakers see "algorithmic fanfare", non-techy people will assume there is some algo in Mastodon messing around with their Home timelines, as this is what they are used to on FB, Insta etc.

If you say trends when you mean trends, then everyone would understand, even the non-techy people :)

(Just had to help a couple of such people on FediTips, which is why I'm posting this issue.)

**Reply by draft13** [reply]
Agree; especially since Mastodon is touted as not having "algorithms." 
![image](https://github.com/user-attachments/assets/59afadc5-ff6f-470b-bcfa-076bd6204bd8)

**Comment by nanos** [comment]
I personally don't like 'fewer algorithmic fanfares' either: 

Firstly, the word 'fanfares' sounds weird in this context, and is not in keeping with the tone of voice of all the other options.

Secondly, and arguably more importantly, it gives no actually meaningful information: what is 'fewer'? which 'algorithms'? The wording leaves me with more questions than answers.

I would agree that 'Hidden from search and trends' is descriptive and meaningful, and focusing on the two most important (to most) aspects of this setting, although maybe others might be able to come up with even better wording.

**Comment by UvixCreative** [comment]
I agree with this. I'm a technical user, and if I had to search "mastodon quiet public," then normal users aren't going to have any clue what that means.

I believe changing the description to "Hidden from search and trends" is far more accessible and easy to understand.

**Reply by JJKraw** [reply]
I'm here to report the exact same thing. Been a techie since the 1970s and I found "fewer algorithmic fanfares" to add zero information to what the feature might mean. Particularly since, as otherwise mentioned, Mastodon is touted as avoiding algorithms.

I Like "Hidden from search and trends". I wouldn't be spending these ten minutes here now if that was the description.

**Comment by chrislo** [comment]
I arrived here by searching "mastodon quiet public" after seeing it as the tooltip to an icon I didn't understand on a user's post. I would have understood "Hidden from search and trends". 

<img width="267" alt="Screenshot 2024-10-28 at 16 21 14" src="https://github.com/user-attachments/assets/eaf7249b-38b5-4450-ba38-95a3d16b4585">

**Comment by mcclure** [comment]
Ran [this poll](https://mastodon.social/@mcc/115238458321421574), and was afterward linked this issue.

How do you feel about these numbers?

<img width="930" height="876" alt="image" src="https://github.com/user-attachments/assets/6de1fe10-9e59-465f-a523-d576ebe7f387" />

This is over a year after this new UI was introduced, so well after everyone has had time to get used to it.

I have always thought "unlisted' was superior wording. It's imprecise, but it *suggests* the idea more expressively.

Someone in the poll suggested "Unindexed", a proposal I haven't seen before, which doesn't capture the idea of the local/federated timelines, but does capture what I consider to be the most important feature of "quiet public" (exclusion from search)

**Reply by trwnh** [reply]
interestingly, recent anecdotal feedback i've received is that "unlisted" is confusing because it functions differently than "unlisted" in many other services. mastodon `unlisted` posts are still visible on your profile and in home timelines and lists. this is in contrast to typical unlisted functionality that hides such items everywhere, making them only accessible by direct URL. the most common point of confusion i've heard is, "why would they be called unlisted if they show up in lists?"

sure, "quiet" is not an ideal qualifier that people understand right away, but it does a better job at conveying that the post is still public (literally `cc: Public`).

as an alternative to "unlisted" or "quiet public", i don't think "unindexed" is quite right either, but perhaps "unpromoted" could work. i know people joked about "low-key public" sounding uncool, but it's not entirely wrong either. personally, i refer to `unlisted` posts as "public" and `public` posts as "discoverable". if we don't want to rename `public`, then heck, call `unlisted` "public 2". that's what it is after all -- a secondary public, where the public is part of the secondary audience instead of the primary audience. (of course, the whole to/cc heuristic has other problems -- see https://github.com/mastodon/mastodon/issues/18212 for that.)

**Comment by diondiondion** [comment]
The original request has been implemented now & the option's description changed to "Hidden from Mastodon search results, trending, and public timelines":

<img width="564" height="321" alt="image" src="https://github.com/user-attachments/assets/43081821-8953-4070-8edb-e7fffeef9792" />
